perf/src/sigverify.rs

```diff
@@ -515,18 +515,11 @@ pub fn shrink_batches(batches: &mut Vec<PacketBatch>) {
 pub fn ed25519_verify_cpu(batches: &mut [PacketBatch], reject_non_vote: bool, packet_count: usize) {
     debug!("CPU ECDSA for {}", packet_count);
     PAR_THREAD_POOL.install(|| {
-        batches
-            .par_iter_mut()
-            .flatten()
-            .collect::<Vec<&mut Packet>>()
-            .par_chunks_mut(VERIFY_PACKET_CHUNK_SIZE)
```

Comment:
removing this, we're effectively making parallelism on whatever the batch size is, instead of this.

Think that is move from 128 -> 64.

Removing allocations here almost certainly better.

Reply:
nevermind, I apparently don't know how flatten works on paralel iterators 😄